libr/core/cmd_open.c

```diff
@@ -570,10 +570,10 @@ static void cmd_open_map(RCore *core, const char *input) {
 	case '.': // "om."
 		map = r_io_map_get (core->io, core->offset);
 		if (map) {
-			core->print->cb_printf ("map: %i fd: %i +0x%"PFMT64x" 0x%"PFMT64x
-				" - 0x%"PFMT64x" ; %s : %s\n", map->id, map->fd,
-				map->delta, map->itv.addr, r_itv_end (map->itv),
-			r_str_rwx_i (map->perm), map->name ? map->name : "");
+			core->print->cb_printf ("%2d fd: %i +0x%08"PFMT64x" 0x%08"PFMT64x
+				" - 0x%08"PFMT64x" %s %s\n", map->id, map->fd,
+				map->delta, map->itv.addr, r_itv_end (map->itv) - 1,
```

Comment:
i think we can make the code more readable if we add some macros in r_io.h like this: 
```c
#define r_io_map_get_from(map) map->itv.addr
#define r_io_map_get_to(map) r_itv_end (map->itv) - 1
...
```

What do you think @condret ? I never liked this weak usage of the RInterval structure, accessing direct methods and substracting values is a source of problems because any change in the behaviour will require updating all the uses of this api.

Reply:
We already have r_itv_begin(&map->itv);, so that maybe probably better usage than acessing fields directly. but having the macros (or static functions in the r_io.h) will make it a bit cleaner maybe. That's up for discussion in here

Reply:
having such functions like r_io_map_begin and r_io_map_end would be great, we should note that down somewhere